### PR TITLE
The traitor uplink encryption key now grants access to the syndicate radio channel

### PR DESCRIPTION
**Pull request by gurfan**
The Centcomm Encryption Key traitor item (now called Syndicate Encryption Key) now also grants access to the Syndicate radio channel.
This allows the user to communicate with Nuclear Operatives or any other traitors who bought the encryption key.

:t is used to talk over the syndicate channel.

~~The syndicate radio channel is only available on the encryption keys bought from the uplink, so assistants/secoffs who are lucky enough to find a hacked encryption key in maintenance won't hear chatter from nukeops.~~
nevermind they dont spawn in maint

:cl:
 * rscadd: Centcomm Encryption Keys are now called Syndicate Encryption Keys in the Uplink Menu.
 * rscadd: The Syndicate Encryption Keys now grant access to the Syndicate radio channel in addition to all the usual channels.

### CODE REVIEW

**Comment by ghost** [comment]
more of a reason to actually buy it if you ask me, as it stands only the binary one is worth it

**Comment by kane-f** [comment]
nah wait you already did that my b

**Comment by Eneocho** [comment]
Make sure that it adds a note to the player mentioning the new radio channel if purchased.
Also, can't wait for the eventual powergaming against the syndie that accidentally says ":t" on common.

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -430,8 +430,8 @@ var/list/uplink_items = list()
 	cost = 5
 
 /datum/uplink_item/device_tools/cipherkey
-	name = "Centcomm Encryption Key"
-	desc = "A key that, when inserted into any radio headset, allows you to listen to and talk on all known Nanotrasen radio channels using their respective communications keys. Screwdriver to replace encryption keys not included."
+	name = "Syndicate Encryption Key"
+	desc = "A key that, when inserted into any radio headset, allows you to listen to and talk on all known Nanotrasen radio channels using their respective communications keys. Access to the Syndicate radio channel is also granted. Screwdriver to replace encryption keys not included."
```
Comment (Kurfursten) [inline code/datums/uplink_item.dm]: Normally headsets show you the relevant key when examined but that doesn't work in this case. Please add "Use :t to speak on Syndicate radio" or something like that to the description.

**Pull request merged**